src/ResourceManagement/AzureBackup/RecoveryServicesBackupManagement/Generated/IProtectedItemOperations.cs

```diff
@@ -150,7 +150,8 @@ public partial interface IProtectedItemOperations
         /// <returns>
         /// The definition of a OperationStatusResponse.
         /// </returns>
-        Task<GetOperationStatusResponse> GetOperationStatusAsync(string resourceGroupName, string resourceName, string fabricName, string containerName, string protectedItemName, string operationId, CustomRequestHeaders customRequestHeaders, CancellationToken cancellationToken);
+
+        Task<BackUpOperationStatusResponse> GetOperationStatusAsync(string resourceGroupName, string resourceName, string fabricName, string containerName, string protectedItemName, string operationId, CustomRequestHeaders customRequestHeaders, CancellationToken cancellationToken);
```

Comment:
Check if this change is related to your corresponding hydra change. Looks like this change is with satish change